README.md

```diff
@@ -224,6 +224,15 @@ Finally, let's make sure owership of protocol assets has been succsessfully tran
 yarn verify:ownership:transfer
 ```
 
+### ArbitrumFoundationVestingWallet Deployer
 
+#### Deploy
+- Set `ARB_URL` and `ARB_KEY` env vars
+- Set FoundationWalletDeploymentConfig variables (for target chain) in [config file](./scripts/foundation-wallet-deployment/config.ts)
+-  run ```yarn deploy:foundation-wallet ```
 
+#### Verify
```

Comment:
would be good to specify what you're verifying here (ie you're not verifying the contract source code on arbiscan, but verifying the parameters are correct)